qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -483,7 +486,7 @@ def _wait_for_completion(
 
         return self._status in required_status
 
-    def _retrieve_result(self) -> Result:
+    def _retrieve_result(self, refresh: bool = False) -> Result:
```

Comment:
parameter not in the docstring. I'm surprise lint didn't catch this.

Reply:
Found out docstrings are not required for private methods: https://www.python.org/dev/peps/pep-0008/#documentation-strings. As a result, pylint does not enforce them either. Must be why it wasn't caught 😅.